local_dynamo.js

```diff
@@ -0,0 +1,49 @@
+const { DocumentClient } = require('aws-sdk/clients/dynamodb');
+const Promise = require('bluebird');
+const dynamo = require('dynamodb');
+const dynamoLocal = require('dynamodb-local');
+
+const Account = require('./src/db/schemas/account');
+const EmailDomainBlacklist = require('./src/db/schemas/email_domain_blacklist');
+const IdentityVerificationMethod = require('./src/db/schemas/identity_verification_method');
+const RecoveryMethod = require('./src/db/schemas/recovery_method');
+
+const LOCAL_DYNAMODB_PORT = 7877;
+const TEST_DYNAMODB_PORT = 7879;
+
+function overrideLocalDynamo({ port } = { port: LOCAL_DYNAMODB_PORT }) {
+    dynamo.documentClient(new DocumentClient({
+        convertEmptyValues: true,
+        endpoint: `localhost:${port}`,
+        sslEnabled: false,
+        region: 'local-env'
+    }));
+}
+
+async function initLocalDynamo({ dbPath, port } = {}) {
+    overrideLocalDynamo({ port });
+    await dynamoLocal.launch(port, dbPath, dbPath && ['-sharedDb']);
+
+    await Promise.all([
+        Account,
+        EmailDomainBlacklist,
+        IdentityVerificationMethod,
+        RecoveryMethod,
+    ].map((model) => model.createTableAsync()))
+        .catch(() => {});
+
+    return {
+        port,
+        terminateLocalDynamo: () => {
+            console.warn(`Stopping local DynamoDB instance on port ${port}`);
+            dynamoLocal.stop(port);
```

Comment:
```suggestion
            await dynamoLocal.stop(port);
```
Feels like the `console.warn` below would be incorrect if we didn't `await` here.

Reply:
Turns out it was a synchronous method the whole time: https://github.com/rynop/dynamodb-local/blob/master/index.js#L100

I'll remove the `Stopped local...` logging below to make it a little less verbose though